include/ConformalTracking.h

```diff
@@ -68,7 +70,7 @@ class ConformalTracking : public Processor {
 
   // Cell creation
   KDCluster* extrapolateCell(Cell*, double);
-  void       extendSeedCells(std::vector<Cell*>&, KDTree*, bool, std::vector<KDCluster*>);
+  void       extendSeedCells(std::vector<Cell*>&, KDTree*, bool, const std::vector<KDCluster*>&);
```

Comment:
Just to check, this just presents a copy of the vector being made, no? It still doesn't allow modification?

Reply:
s/presents/prevents/ ?

Reply:
yes, should read prevents!

Reply:
Then yes, it prevents modifications of the object